pages/batch-qrcode.vue

```diff
@@ -160,6 +162,9 @@ onMounted(() => {
 
 async function drawQRCodes () {
   try {
+    if (!csvInput.value.includes(CSV_HEADER)) {
+      csvInput.value = `${CSV_HEADER}\n${csvInput.value}`
+    }
     urlItems.value = csvParse(csvInput.value, {
```

Comment:
Better not alter input value
```suggestion
    let csvInput = cvsInput.value
    if (!csvInput.includes(CSV_HEADER)) {
      csvInput = `${CSV_HEADER}\n${csvInput}`
    }
    urlItems.value = csvParse(csvInput, {
```